Sources/Sentry/SentryOptions.m

```diff
@@ -54,6 +54,10 @@ - (instancetype)init
         _experimentalEnableTraceSampling = NO;
         _enableSwizzling = YES;
 
+        _profilerSampleRateHz = 100;
+        _measureProfilerCost = YES;
+        _profilerEnabled = YES;
```

Comment:
`h`: I'm pretty sure we don't want to enable this per default yet 😁. I'm fine with turning it on in the samples and also in the [integration tests](https://github.com/getsentry/sentry-cocoa/blob/master/.github/workflows/integration-tests.yml). For the integration tests we would need to add a line `options.profilerEnabled = true` to the git patch here https://github.com/getsentry/sentry-cocoa/blob/06f40f9b1c0d9c979f7e0f20ec7563b7f40facaf/scripts/add-sentry-to-vlc.patch#L35

and also to `add-sentry-to-homekit.patch` and `add-sentry-to-alamofire.patch`.

```suggestion
        _measureProfilerCost = NO;
        _profilerEnabled = NO;
```